src/components/card/index.jsx

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line react/prop-types
+export default function Card({ title = '', img }) {
+  return (
+    <div className="shop1">
+      <span className="card-title">{title}</span>
+      <img src={img} alt="картинка" className="card-img" />
```

Comment:
alt attribute should be in english and should represent what you see on this img